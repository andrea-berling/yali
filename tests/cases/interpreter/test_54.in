// This program uses the logical OR operator to print the first value that is truthy
print 91 or true;
print false or 91;
print false or false or true;
print false or false;
print false or false or false;
print false or false or false or false;
