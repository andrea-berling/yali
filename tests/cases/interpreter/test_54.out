91
91
true
false
false
false
